poly-markdown.el

```diff
@@ -65,7 +65,7 @@
   :tail-mode 'host)
 
 (define-innermode poly-markdown-inline-code-innermode
-  :head-matcher (cons "[^`]\\(`\\)[[:alnum:]+-]" 1)
+  :head-matcher (cons "[^`]\\(`\\)[[:alnum:]([{&*+-]" 1)
   :tail-matcher (cons "\\(`\\)[^`]" 1)
```

Comment:
I guess it could then be replaced with non-whitespace then.

Reply:
Be careful with it. It tends to break easily. I believe it's a big in the core crate

Reply:
Yerh. It does break badly indeed. 

I now see that there are a number of issues with this inline innermode. I think I will remove it for time being.